package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "babel-node src/index.js",
+    "debug": "babel-node src/index.js 20",
```

Comment:
`loglevel` can be config in `.ENV`

Reply:
.env file is excluded for git commit,  if we want to use it, will everyone need to config it on their own?